pkg/cmd/server/origin/master.go

```diff
@@ -1063,6 +1065,35 @@ func (c *MasterConfig) RunSecurityAllocationController() {
 	controller.Run()
 }
 
+func (c *MasterConfig) BootstrapSecurityContextConstraints() {
+	sccList, err := c.KubeClient().SecurityContextConstraints().List(labels.Everything(), fields.Everything())
+	if err != nil {
+		glog.Fatalf("Unable to initialize security context constrints: %v", err)
+	}
+	if len(sccList.Items) > 0 {
+		return
+	}
+	glog.Infof("No security context constraints detected, adding defaults")
+	scc := &kapi.SecurityContextConstraints{
+		ObjectMeta: kapi.ObjectMeta{
+			Name: "scc-default",
+		},
+		AllowPrivilegedContainer: true,
```

Comment:
10-4.  This is just to get the mechanics right.  The secured model will create SCCs that are very close to https://github.com/openshift/origin/pull/2609#issuecomment-108608800 once the dependent PRs land.